en/docs/install-and-setup/setup/single-node/configuring-a-single-node.md

```diff
@@ -107,12 +107,9 @@ In this case, let's use `api.am.wso2.com` as the hostname:
 
 ## Step 6 - Configure API-M Analytics
 
-If you wish to view reports, statistics, and graphs related to the APIs deployed in the WSO2 API Manager, you need to 
-configure API-M Analytics. If not, you can **skip this step**.
+API Manager Analytics is delivered via the API Manager Analytics cloud solution. You need to configure the API-M Gateway needs to publish analytics data into the cloud.
 
-Follow the [Configuring API-M Anlaytics - Quick Setup]({{base_path}}/observe/api-manager-analytics/configuring-apim-analytics/#quick-setup) to configure API-M Analytics in a development setup and, follow 
-[Configuring API-M Analytics - Standard Setup]({{base_path}}/observe/api-manager-analytics/configuring-apim-analytics/#standard-setup) 
-to configure API-M Analytics in a production setup.
+See the instructions on [configuring the synapse Gateway]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-synapse-gateway) with the cloud-based analytics solution.
```

Comment:
We don't use the term "synapse gateway" now. @tishan89 should we just say "gateway"?

Reply:
Name is still under discussion afaik. Ive asked Sanjeewa and will confirm with u shortly

Reply:
Lets use API Gateway